cost-analyzer/templates/query-service-deployment-template.yaml

```
@@ -61,7 +61,8 @@ spec:
         {{- end }}
       initContainers:
         - name: config-db-perms-fix
-          image: busybox
+          image: "{{ .Values.kubecostDeployment.queryService.busyBoxImage.repository }}:{{ .Values.kubecostDeployment.queryService.busyBoxImage.tag }}"
+          imagePullPolicy: "{{ .Values.kubecostDeployment.queryService.busyBoxImage.pullPolicy }}"
```

Because I'm assuming this config will be less frequently modified, what are your thoughts on removing defaults in the `values.yaml` and putting them here instead?

```suggestion
          image: "{{ .Values.kubecostDeployment.queryService.initImage.repository | default busybox }}:{{ .Values.kubecostDeployment.queryService.initImage.tag | default stable }}"
          imagePullPolicy: "{{ .Values.kubecostDeployment.queryService.initImage.pullPolicy | default IfNotPresent}}"
```

---

LGTM. You want to commit these changes Thomas? Sean- please let us know if you have any concerns.

---

LGTM. Changes committed as proposed.